site/public/js/ta-grading-mark.js

```diff
@@ -176,6 +226,23 @@ function updateMarksOnPage(num, background, min, max, precision, gradeable_id, u
         var marks = $('[name=mark_'+num+']');
         for (var x = 0; x < marks.length; x++)
             marks[x].remove();
```

Comment:
Can we put `{ }` around the for loop body, especially if we're going to then have a separate block scope immediately following this line?